include/ImNodeFlow.h

```diff
@@ -204,7 +204,7 @@ namespace ImFlow
          * @param right Pointer to the input Pin of the Link
          * @param inf Pointer to the Handler that contains the Link
          */
-        explicit Link(Pin* left, Pin* right, ImNodeFlow* inf) :m_left(left), m_right(right), m_inf(inf) {}
+        explicit Link(Pin* left, Pin* right, ImNodeFlow* inf) : m_inf(inf), m_left(left), m_right(right) {}
```

Comment:
I prefer m_inf to be last because the main parameters are left and right. So I prefer the order: left, right, inf. If somewhere this order isn't followed, that's were it needs changing

Reply:
As commented bellow, I reverted the changes in the initializer list and instead changed the order of the member variable declarations to match the order in the constructor.